src/utils/datadog.ts

```diff
@@ -0,0 +1,74 @@
+import { datadogRum } from '@datadog/browser-rum';
+
+/**
+ * Gets the configuration values for datadog based on the environment.
+ * It returns null if the environment is not staging or production.
+ * @param {string} environment - The environment to get the configuration values for.
+ * @example getConfigValues('staging');
+ * @returns {object|null} - The configuration values for datadog.
+ * **/
+const getConfigValues = (environment: string) => {
+    if (environment === 'production') {
+        return {
+            serviceName: 'deriv-bot',
+            dataDogVersion: `deriv-bot-${process.env.REF_NAME}`,
+            dataDogSessionReplaySampleRate: Number(process.env.DATADOG_SESSION_REPLAY_SAMPLE_RATE ?? 1),
+            dataDogSessionSampleRate: Number(process.env.DATADOG_SESSION_SAMPLE_RATE ?? 10),
+            dataDogEnv: 'production',
+        };
+    } else if (environment === 'staging') {
+        return {
+            serviceName: 'staging-dbot.deriv.com',
+            dataDogVersion: `deriv-dbot-staging-v${process.env.REF_NAME}`,
```

Comment:
```suggestion
            dataDogVersion: `v${process.env.REF_NAME}`,
```
nit: you dont need to make it too long. as you can filter with serviceName first